docs/portal.md

```diff
@@ -416,6 +416,49 @@ api.portal.set_registry_record('field_one', 'new value', interface=IMyRegistrySe
 %     'new value'
 % )
 
+(portal-get-vocabulary-example)=
+
+## Get vocabulary
+
+To get a vocabulary by name, use {meth}`api.portal.get_vocabulary`.
```

Comment:
```suggestion
To get a vocabulary by name, use {func}`api.portal.get_vocabulary`.
```
But can be done later, I do not want to block this one!

Reply:
Let's do it now. Good catch!